ocf_datapipes/training/windnet.py

```diff
@@ -123,7 +123,6 @@ def __init__(self, filenames: List[str], keys: List[str]):
 
     def __iter__(self):
         """Iterate through each filename, loading it, uncombining it, and then yielding it"""
-        import numpy as np
```

Comment:
@jacobbieker just checking this import of numpy here didn't have some specific use and can be removed?